packages/trace-viewer/src/ui/attachmentsTab.tsx

```diff
@@ -66,6 +66,6 @@ function attachmentURL(traceUrl: string, attachment: {
   body?: string;
 }) {
   if (attachment.sha1)
-    return 'sha1/' + attachment.sha1 + '?trace=' + traceUrl;
+    return 'sha1/' + attachment.sha1 + '?trace=' + encodeURIComponent(traceUrl);
```

Comment:
I feel like all the trace viewer changes are unrelated, let's move them to a separate PR?

Reply:
yup https://github.com/microsoft/playwright/pull/23893.